DependencyInjection/DunglasActionExtension.php

```diff
@@ -34,7 +34,8 @@ public function load(array $configs, ContainerBuilder $container)
 
         $directoryList = [];
         foreach ($config['directories'] as $pattern) {
-            list($classes, $directories) = $this->getClasses($kernelRootDir.DIRECTORY_SEPARATOR.$pattern);
+            $directory = (DIRECTORY_SEPARATOR !== substr($pattern, 0, 1)) ? $kernelRootDir.DIRECTORY_SEPARATOR.$pattern : $pattern;
```

Comment:
why so? I don't mind but I don't see why it should be `/` here and `DIRECTORY_SEPARATOR` above